Notebooks/OOPS_TMT_Data_Processing_And_QC.Rmd

```diff
@@ -143,10 +181,11 @@ Now we have filtered our PSM-level quantification, summarised to protein-level a
 
 We save the object to disk so we can read it back into memory when we need it
 ```{r}
-saveRDS(psm_filt_sn_int_missing, here('OOPS-TMT-HepG2.C3A/Results/input_psm_filt.rds'))
-saveRDS(protein_norm, here( 'OOPS-TMT-HepG2.C3A/Results/input_tmt_protein.rds'))
+saveRDS(psm_filt_sn_int_missing, here('Results/input_psm_filt.rds'))
```

Comment:
`here` will start at the top of the Rproj. Not sure how it was working before as that path shouldn't have existed :man_shrugging: